### ISSUE TITLE
mbedtls config with only curve25519

### ISSUE THREAD

**Issue by pegahnikbakht** [ARMmbed/mbedtls#2901]
### Description
I want to config mbedtls with curve25519 only but I get the following error? 
Did you add the support or still it has problem? 
---------------------------------------------------------------
## Bug
../library/libmbedcrypto.a(ecp.o): In function `mbedtls_ecp_muladd_restartable':
ecp.c:(.text+0x142b): undefined reference to `ecp_add_mixed'
ecp.c:(.text+0x143e): undefined reference to `ecp_normalize_jac'
collect2: error: ld returned 1 exit status
Makefile:102: recipe for target 'test_suite_ssl' failed
make[1]: *** [test_suite_ssl] Error 1
Makefile:21: recipe for target 'tests' failed
make: *** [tests] Error 2

**Comment by RonEld** [comment]
Hi @pegahnikbakht 
Thank you for reporting this issue!
I believe this is a duplicate of #941.
Does #2019 Fix your issue?

**Comment by pegahnikbakht** [comment]
@RonEld You mean using this branch https://github.com/gilles-peskine-arm/mbedtls/tree/build_with_only_montgomery_curves-conditional_mul_add-march ?

**Comment by RonEld** [comment]
@pegahnikbakht 
Yes, I meant whether the PR, which is implemented in the branch you referenced fixes your issue.

**Comment by gilles-peskine-arm** [comment]
Indeed #2019 should fix this, but unfortunately it needs quite a bit of rework (lots of merge conflicts now) and I'm afraid I don't have time for this rework at the moment.

**Comment by RonEld** [comment]
Thanks for confirmation.
CLosing this issue as a duplicate of #941
